src/menuList.js

```diff
@@ -81,21 +99,45 @@ const MenuList = ({ user, logout, addSuggestion }) => {
   if (user) {
     return (
       <>
+        { !open ?
         <List>
-          {menuConfigs.map(menuConfig => renderMenuItem(user, menuConfig))}
-          <ListItem button onClick={() => setOpen(true)}>
-            <ListItemIcon>
-              <RecordVoiceOverIcon />
-            </ListItemIcon>
-            <ListItemText>건의사항</ListItemText>
-          </ListItem>
+          {menuConfigs.map(menuConfig => renderMenuItemTooltip(user, menuConfig))}
```

Comment:
Jin, see if you can use `menuConfigs` to include tooltip text and use it in `renderMenuItem`.